docs/nodes/compute/advanced/local-testing.md

```diff
@@ -0,0 +1,99 @@
+# Running Instances on the Local Environment
+
+Aleph.im supports two popular and high-performance hypervisors for virtualization: Firecracker and Qemu. This guide outlines the steps to run instances on your local machine for development and testing purposes.
+
+## 0. Prerequisites
+
+- **SSH Keys:** Ensure you have a valid SSH key pair set up. You'll need the public key to enable SSH access to the instances.
+
+## 1. Configuration
+
+### **Configure Allocation Token and SSH Keys:**
+
+- Edit the `/etc/aleph/supervisor.env` file and add the following lines:
+
+  ``` bash
+  # Encrypted allocation token hash in sha1 format
+  ALEPH_VM_ALLOCATION_TOKEN_HASH=YOUR_ALLOCATION_TOKEN_HASH
+
+  # List of authorized SSH public keys (replace "YOUR_SSH_PUBLIC_KEY" with your actual key)
+  ALEPH_VM_DEVELOPER_SSH_KEYS=["YOUR_SSH_PUBLIC_KEY"]
+  ```
+
+- **Important:** Replace `YOUR_ALLOCATION_TOKEN_HASH` with your actual encrypted allocation token and `YOUR_SSH_PUBLIC_KEY` with your public SSH key.
+
+
+### **Enable Developer SSH Keys (Optional):**
+
+By default, SSH access to instances is only enabled to the user that creates the VM for security reasons. To enable it for development, you have two options:
+
+#### a) Modifying the Systemd Service File:
+
+1. Locate the Aleph VM orchestrator service file, typically found at `/etc/systemd/system/aleph-vm-supervisor.service`.
+2. Edit the file and add the `--developer-ssh-keys` argument to the `ExecStart` line. It should look something like this:
+
+   ``` bash
+   ExecStart=python3 -m aleph.vm.orchestrator --print-settings --very-verbose --developer-ssh-keys
+   ```
+
+3. Reload the systemd configuration and restart the service:
+
+   ``` bash
+   sudo systemctl daemon-reload
+   sudo systemctl restart aleph-vm-supervisor
+   ```
+
+#### b) Running the Orchestrator Manually:
+
+You can simply append the `--developer-ssh-keys` argument when running the orchestrator manually:
+
+ ``` bash
+ python3 -m aleph.vm.orchestrator --print-settings --very-verbose --developer-ssh-keys
+ ```
+
+## 2. Launch Instance
+
+Use the following `curl` command to launch an instance locally:
+
+ ``` bash
+ curl --retry 5 --max-time 10 --fail -X POST -H "Content-Type: application/json" \
```

Comment:
Why retry 5 times with a timeout when testing locally ? We are not waiting for a DigitalOcean droplet to boot...

```suggestion
 curl --verbose --include -X POST -H "Content-Type: application/json" \
```